packages/ng/src/app/select/select.module.ts

```diff
@@ -12,13 +15,16 @@ import { LuPopoverModule } from '../popover/popover.module';
 @NgModule({
 	imports: [
 		CommonModule,
+		FormsModule,
+		BrowserModule,
 		OverlayModule,
 		LuPopoverModule,
 		LuEmptyModule,
```

Comment:
does it need the `empty` module ? i dont think so

Reply:
Maybe, indeed, i will check that 👍